### PR TITLE
feat: Replace new Vue() with custom event bus (no-changelog)

### PR DESCRIPTION
**Pull request by alexgrozav**

### CODE REVIEW

File: packages/editor-ui/src/components/TagsDropdown.vue [hunk 1]
```diff
@@ -109,7 +118,7 @@ export default mixins(showMessage).extend({
 		}
 
 		if (this.$props.eventBus) {
-			this.$props.eventBus.$on('focus', () => {
+			this.$props.eventBus.on('focus', () => {
```
Comment (OlegIvaniv) [inline packages/editor-ui/src/components/TagsDropdown.vue:121]: There's a lot of cases where we use `this.$props` in the codebase for no apparent reason, which breaks types suggest. Maybe we could also replace these?

**Review by OlegIvaniv — APPROVED** [review]
Nice job! One step closer to Vue 3 👣

File: packages/editor-ui/src/components/TagsDropdown.vue (hunk 1 above)
Comment (OlegIvaniv) [reply packages/editor-ui/src/components/TagsDropdown.vue:121]: Will be tackled as tech debt — [N8N-6113](https://linear.app/n8n/issue/N8N-6313/replace-thisdollarpropspropname-with-thispropname)

**Comment by janober** [comment]
Got released with [`n8n@0.224.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.224.0)

**Pull request merged**